### ISSUE TITLE
StructureFromMotion ERROR

### ISSUE THREAD

**Issue by jadapa** [alicevision/meshroom#331]
Hello,
When I start the process everything works fine until it reaches the node StructureFromMotion. The node goes red and the process stops.
The photos were taken with Xiaomi Mi A2 Lite.
I also get this error in the console on startup: Failed to download scene at QUrl("")

**Comment by fabiencastan** [comment]
Hi,
If your dataset is not connected enough, the StructureFromMotion may fail to find a solution.
You can test with this dataset: https://github.com/alicevision/dataset_monstree to validate that everything works fine on your computer.
The warning at startup is not an issue.
Best,

**Comment by jadapa** [comment]
Hi,
I haven't tested the dataset yet, but the error may be there because my phone is dual-camera and both cameras are different, so the database may know one of them, but not the other one. Couldn't this be the problem?

**Comment by fabiencastan** [comment]
Yes, if there is a significant error on the sensor size, the SfM step will fail.

**Comment by jadapa** [comment]
Ok. So, where to find both cameras? I have found the sensor "Sony IMX486 Exmor RS" here: https://www.kimovil.com/en/xiaomi-mi-a2-lite/camera and I've set the sensor width to 6.2 in the database according to this Wikipedia site:  https://en.wikipedia.org/wiki/Exmor
So the line in the database looks like this: Sony;IMX486 Exmor RS;6.2.
I've also tried these:
Sony;IMX486 Exmor;6.2
Sony;IMX486;6.2
Sony;IMX486-Exmor-RS;6.2
Xiaomi;Mi A2 Lite; 6.2
Xiaomi;Mi A2;6.2
Xiaomi;A2 Lite;6.2
Xiaomi;A2;6.2
But none of them work
